src/Microsoft.Android.Runtime.NativeAOT/Java.Interop/JreRuntime.cs

```diff
@@ -0,0 +1,180 @@
+// Originally from: https://github.com/dotnet/java-interop/blob/dd3c1d0514addfe379f050627b3e97493e985da6/src/Java.Runtime.Environment/Java.Interop/JreRuntime.cs
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Microsoft.Android.Runtime;
+
+namespace Java.Interop {
+
+	struct JavaVMInitArgs {
+		#pragma warning disable CS0649 // Field is never assigned to;
+		public  JniVersion                      version;    /*				 use JNI_VERSION_1_2 or later */
+
+		public  int                             nOptions;
+		public  IntPtr /* JavaVMOption[] */     options;
+		public  byte                            ignoreUnrecognized;
+		#pragma warning restore CS0649
+	}
+
+	class JreRuntimeOptions : JniRuntime.CreationOptions {
```

Comment:
We should rename this `NativeAotRuntimeOptions`, trim out anything that we don't need or can't support, such as `Options`, `ClassPath`, `IgnoreUnrecognizedOptions`, etc.  This would in turn remove `JvmLibraryHandler` entirely (don't need it!), etc.